crates/starknet-os/src/hints/tests.rs

```diff
@@ -278,4 +282,136 @@ pub mod tests {
             );
         }
     }
+
+    #[test]
+    fn test_fill_holes_in_rc96_segment() {
+        let mut vm = VirtualMachine::new(false);
+        vm.set_fp(1);
+        vm.add_memory_segment();
+        vm.add_memory_segment();
+
+        let mut exec_scopes = ExecutionScopes::new();
+        let ids_data = ids_data![vars::ids::RANGE_CHECK96_PTR];
+        let ap_tracking = ApTracking::default();
+        let constants = HashMap::new();
+
+        let mut rc96_segment = vm.add_memory_segment();
+        rc96_segment.offset = 10;
+        insert_value_from_var_name(vars::ids::RANGE_CHECK96_PTR, rc96_segment, &mut vm, &ids_data, &ap_tracking)
+            .expect("insert_value_from_var_name");
+
+        let rc96_base = with_offset(rc96_segment, 0);
+        vm.insert_value(rc96_base, Felt252::THREE).expect("insert value at base");
+        for i in 1..rc96_segment.offset {
+            let address = with_offset(rc96_segment, i);
+            assert_eq!(vm.get_maybe(&address), None);
```

Comment:
In `fill_holes_in_rc96_segment()` we do `vm.insert_value(address, Felt252::ZERO)?;` for each hole, so shouldn't we expect to get `Felt252::ZERO` here instead of `None`?

Reply:
Nevermind, this is before calling the hint :)